annotate.el

```diff
@@ -627,7 +627,9 @@ an overlay and it's annotation."
 
 (defun annotate-load-annotation-data ()
   "Read and return saved annotations."
+  (setq local-annotate-file annotate-file)
   (with-temp-buffer
+    (setq-local annotate-file local-annotate-file)
```

Comment:
It's probably nicer to replace the `annotate-file` in the next two lines with the new `local-annotate-file` instead of `setq-local`ing the variable.